index.js

```diff
@@ -171,24 +171,22 @@ class USHINBase {
   }
 
   async getPointsForMessage({ main, shapes }) {
-    const allPoints = new Set();
+    const pointIds = [main, ...Object.values(shapes).flat()];
```

Comment:
Maybe we should just use the `allPoinnts` array from the message instead of `main` and `shapes`? :P